keyboards/yandrstudio/sp60/rev2/rev2.h

```diff
@@ -0,0 +1,34 @@
+// Copyright 2022 Y&R-Biu (@jiaxin96)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include "sp60.h"
+
+#define LAYOUT_60_tsangan_split_rshift( \
```

Comment:
Layout macros should be moved to into.json by adding `"matrix": [row,col]` to each key, and this file can be deleted.